app/services/prison_visits/client.rb

```diff
@@ -43,7 +43,7 @@ def request(method, route, params)
 
       response = @connection.request(options)
 
-      JSON.parse(response.body)
+      response.body.present? ? JSON.parse(response.body) : nil
```

Comment:
I thought of that. I find returning no body more explicit than returning an empty object that is not used by the client.

I don't feel particularly strong about either approach as long as we are consistent with the approach later.